src/main/java/dukebot/command/AliasCommand.java

```diff
@@ -0,0 +1,53 @@
+package dukebot.command;
+
+import dukebot.exception.DukeException;
+import dukebot.storage.AppStorage;
+import dukebot.storage.Storage;
+import dukebot.tasklist.TaskList;
+import dukebot.ui.LineName;
+import dukebot.ui.Ui;
+
+import java.util.HashMap;
+
+/**
+ * Changes phrase mapping of command.
+ */
+public class AliasCommand extends Command {
+    private String[] inpArr;
```

Comment:
Perhaps consider a better name for this variable?

Reply:
Variable is just the input split by space and placed into an array, hence inputArray or inpArr.
Not sure what better name there can be.

Reply:
`inputArray` is probably easier to understand as compared to `inpArr`...